javascript/javascript-core/functions-ii/iife-pattern.md

```diff
@@ -98,3 +97,46 @@ What's the syntax of an **IIFE** ?
 - `name`
 - `call`
  
+---
+
+## Footnotes
+
+[1: IIFE Syntax]
+
+IIFEs have two ways of writing them:
+
+```js
+(function () {
+  console.log('this works');
+}());
+// ^ invocation parenthesis are inside the outer parenthesis
+
+(function () {
+  console.log('so does this');
+})();
+// ^ invocation parenthesis are outside of the outer parenthesis
+```
+
+This code would output (check out this [StackOverflow thread](https://stackoverflow.com/questions/3384504/location-of-parenthesis-for-auto-executing-anonymous-javascript-functions) for more details):
```

Comment:
This feels broken? The image is missing?

Reply:
I guess the "This code would output" part should go above line 122

Reply:
> This feels broken? The image is missing?

the image was 2 lines down, but I can see how this can be confusing so I've updated it :+1: